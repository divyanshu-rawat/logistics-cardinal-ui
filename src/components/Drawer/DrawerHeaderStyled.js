import styled from 'styled-components';

const DrawerHeaderStyled = styled('div')`
  min-height: 100px;
  display: flex;
  align-items: center;
  padding: ${({ theme }) => theme.composedSpacings.byte};
`;

export default DrawerHeaderStyled;
