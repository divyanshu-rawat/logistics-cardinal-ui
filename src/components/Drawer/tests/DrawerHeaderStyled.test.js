import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import DrawerHeaderStyled from '../DrawerHeaderStyled';
import Theme from '../../../themes/rooster';

const renderedComponent = () => shallow(<DrawerHeaderStyled theme={Theme} />);

describe('<DrawerHeaderStyled />', () => {
  it('should render the component', () => {
    const tree = toJson(renderedComponent());
    expect(tree).toMatchSnapshot();
  });
});
