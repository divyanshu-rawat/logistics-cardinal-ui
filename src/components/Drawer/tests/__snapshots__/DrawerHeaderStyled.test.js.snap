// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DrawerHeaderStyled /> should render the component 1`] = `
<div
  className="DrawerHeaderStyled-fVieIv ljGyyO"
/>
`;
